source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -749,6 +754,12 @@ response to the current ServerDescription's topologyVersion:
 See `Replacing the TopologyDescription`_ for an example implementation of
 topologyVersion comparison.
 
+serviceId
+````````
+
+MongoDB 5.0 and later, as well as any mongos-like service include a ``serviceId``
```

Comment:
Comma between `service` and `include`.  Without the comma, I totally misinterpreted this sentence.

Reply:
Fixed.